src/index.html

```diff
@@ -1,13 +1,76 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta
+      name="viewport"
+      content="
+        width=device-width,
+        user-scalable=no,
+        initial-scale=1.0,
+        maximum-scale=1.0,
+        minimum-scale=1.0
+      "
+    >
+    <title>Product cards</title>
+    <link
+      rel="stylesheet"
+      href="./styles/index.css"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    >
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
+  </head>
+  <body class="page__body">
+    <div class="container">
+      <div class="imac imac--position" data-qa="card">
+        <img
+          class="product-image product-image--position"
+          src="./images/imac.jpeg"
+          alt="iMac image"
+        >
+
+        <div class="full-name full-name--position">
+          Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+        </div>
+
+        <div class="product-code product-code--position">
+          Код товара: 195434
+        </div>
+
+        <div class="rating rating--position">
```

Comment:
Not BEM

Reply:
Try to implement something like this
```
card
	card__image
	card__title
	...
	card__rating
		...
		card__rating-feedback
		...
```
`card` is a BEM-block here, and everything inside – its BEM-elements

Reply:
Also, make sure you don't create a lot of BEM-blocks, there are only two BEM-blocks you need to create – `card` and `stars`
NB: each BEM-block requires a separate file with a correspondent name